swagger.yml

```diff
@@ -1136,6 +1136,8 @@ components:
                     type: string
                 TurnEndpoint:
                     type: string
+                PersistentKeepAlive:
```

Comment:
swagger.yml is a generated file - should not be updated manually

Reply:
https://github.com/gravitl/netmaker/pull/2604/commits/5ca35fea91a7af804b7eb87262360b0a435f6406